coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift

```diff
@@ -6,15 +6,20 @@
 //
 
 import SwiftUI
+import SwiftData
 
 struct DetailsScreen: View {
     @Environment(\.dismiss) var dismiss
+    @Environment(\.modelContext) var modelContext
+    @Query var tasks: [Task]
```

Comment:
```suggestion
    @Query var tasks: [Task]
```

I had not seen this before. 😀